hypervisor/Cargo.toml

```diff
@@ -16,7 +16,7 @@ epoll = "4.3.1"
 thiserror = "1.0.30"
 libc = "0.2.112"
 log = "0.4.14"
-kvm-ioctls = { version = "0.11.0", optional = true }
+kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "main", optional = true }
```

Comment:
You don't need to do that, the crate is already patched from the root `Cargo.toml` https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/Cargo.toml#L39
You simply need to run `cargo update -p kvm-ioctls`, which will update `Cargo.lock`.